src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,594 @@
+| symbol               | string           | Token symbol.                                                                                    |
+| token_uri            | string           | Token URI.                                                                                       |
+| metadata             | string           | Token metadata.                                                                                  |
+| last_token_uri_sync  | string           | Last time the token URI was synced.                                                              |
+| last_metadata_sync   | string           | Last time the token metadata was synced.                                                         |
+| minter_address       | string           | Minter address.                                                                                  |
+| possible_spam        | boolean          | Possible spam.                                                                                   |
+| uri_meta             | object           | A TokenMetadata object. The contents of this object varies, but will include values for `image`, `token_name`, `description`, `attributes` or `animation_url` (which might be `null`). |
+
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+```json
+{
+  "mmrpc": "2.0",
+  "result": {
+    "chain": "BSC",
+    "token_address": "0xfd913a305d70a60aac4faac70c739563738e1f81",
+    "token_id": "214300044414",
+    "amount": "1",
+    "owner_of": "0xf622a6c52c94b500542e2ae6bcad24c53bc5b6a2",
+    "token_hash": "125f8f4e952e107c257960000b4b250e",
+    "block_number_minted": 25810308,
+    "block_number": 28056721,
+    "contract_type": "ERC721",
+    "collection_name": "Binance NFT Mystery Box-Back to Blockchain Future",
+    "symbol": "BMBBBF",
+    "token_uri": "https://public.nftstatic.com/static/nft/BSC/BMBBBF/214300044414",
+    "metadata": "{\"image\":\"https://public.nftstatic.com/static/nft/res/4df0a5da04174e1e9be04b22a805f605.png\",\"name\":\"Nebula Nodes\",\"description\":\"Interchain nodes continuously exchange blockchain data amongst each other, supporting blockchain interoperability.\"}",
+    "last_token_uri_sync": "2023-02-19T19:12:09.471Z",
+    "last_metadata_sync": "2023-02-19T19:12:18.080Z",
+    "minter_address": "0xdbdeb0895f3681b87fb3654b5cf3e05546ba24a9",
+    "possible_spam": false,
+    "uri_meta": {
+      "image": "https://public.nftstatic.com/static/nft/res/4df0a5da04174e1e9be04b22a805f605.png",
+      "token_name": "Nebula Nodes",
+      "description": "Interchain nodes continuously exchange blockchain data amongst each other, supporting blockchain interoperability.",
+      "attributes": null,
+      "animation_url": null
+    }
+  },
+  "id": null
+}
+```
+</CollapsibleSection>
+
+
+## Withdraw NFTs {{label : 'withdraw_nft', tag : 'API-v2'}}
+
```

Comment:
I think from this line you should to specify two types of withdraw nft requests.
for erc721 tokens and for erc1155 tokens.